doc/source/intro.rst

```diff
@@ -13,6 +13,13 @@ Install the ``ansys-dpf-composites`` package with ``pip``:
 
     pip install ansys-dpf-composites
 
+
+To install the optional requirements used in the examples, add the ``[examples]`` suffix when installing the package:
+
+.. code::
+
+    pip install ansys-dpf-composites[examples]
```

Comment:
Alternative: We could modify the instructions above to always include the `[examples]` extra. 

Benefit: less confusion when the optional requirements are missing b/c the user followed the above instructions only.

Drawback:
- slightly less visible how to get a minimal install (no big deal IMO, users which need that probably know the syntax for optional requirements)
- doesn't work in combination with the specific version option below (since only the next release will have the `[examples]` extra).

Reply:
@greschd Can you explain what "optional requirements" means?
- Are they necessary in the example?
- What's the benefit of installing them?

Reply:
@nshum4 they are optional since PyDPF Composites can be used without them. In other words, they are not directly used by PyDPF Composites.
However, we use them in the examples[1]. So, a user who wants to follow along with the examples will likely want to use this method of installing, s.t. they don't have to manually find out which libraries are used in the examples.

Conversely, if a user wants a _minimal_ install (only the minimum needed to run their analysis), they might not want these dependencies.

[1] some of them are used in all examples, some only in a few.

Reply:
@greschd Understood, thank you! I'm neutral to both ideas. I don't think it would be confusing to add the note about _examples_. However, the phrase "optional requirements" is contradictory. Is this often used in software terms, or can we rewrite it?

Reply:
Let's use optional _dependencies_ instead.

Reply:
Yes, I like that idea! Thank you!